Crypto/samples/genrsakey/CMakeLists.txt

```diff
@@ -1,2 +1,4 @@
 add_executable(genrsakey src/genrsakey.cpp)
 target_link_libraries(genrsakey PUBLIC Poco::Crypto Poco::Util Poco::XML)
+
+POCO_ENABLE_EXE_WMAIN(genrsakey)
```

Comment:
@Bjoe can you please confirm this is the right thing to do

Reply:
See my review comment...